public_html/launch.php

```diff
@@ -511,11 +554,30 @@ function build_form_param($param_id, $param, $is_required){
     simply run the following command and follow the prompts:</p>
 <pre>nf-core launch --id <?php echo $cache_id; ?></pre>
 
+<h3>Launch using Nextflow Tower</h3>
+<?php if(substr($cache['pipeline'], 0, 8) == 'nf-core/'){ ?>
+<p>Clicking the button below will take you to the Nextflow Tower launch page with all parameters set, ready for launch
+(requires a Nextflow Tower account).</p>
+<form method="get" action="https://scratch.staging-tower.xyz/launch" target="_blank" class="mb-3">
```

Comment:
```suggestion
<form method="get" action="https://tower.nf/launch" target="_blank" class="mb-3">
```

Reply:
Updated the URL ready for a direct commit if required. Just need one of you HTML peeps to approve.